controller/commentsController.js

```diff
@@ -47,3 +48,42 @@ exports.flagComment = async (req, res, next) => {
     next(error);
   }
 };
+
+exports.updateComment = async (req, res, next) => {
+  const comment_id = req.params.commentId;
+  const commentBody = req.body.content;
+  const owner = req.body.ownerId;
```

Comment:
I'd also suggest updating this.

```suggestion
  const ownerId = req.body.ownerId;
```

Reply:
Done